package.json

```diff
@@ -35,11 +35,11 @@
   "dependencies": {
     "@babel/core": "7.17.8",
     "@babel/generator": "7.17.7",
-    "@babel/parser": "7.18.9",
+    "@babel/parser": "^7.20.5",
     "@babel/traverse": "7.17.3",
     "@babel/types": "7.17.0",
     "javascript-natural-sort": "0.7.1",
-    "lodash": "4.17.21"
+    "lodash": "^4.17.21"
```

Comment:
Can we not use the fixed version for the deps like`7.20.5`  ?

Reply:
Since `lodash` is the dependency of this package. Users who install this package will install `lodash` too, so in my opinion it is best to be as flexible as possible by using `^` or `~`